src/Gurobi.cpp

```diff
@@ -59,22 +59,134 @@ const VectorXd& GurobiCommon::result() const
 	return X_;
 }
 
+int GurobiCommon::warmStart() const
```

Comment:
Would it be possible to return a more explicit warmStatus ? Some kind of enum like so maybe:
enum WarmStatus
{
     DEFAULT= -1,
     PRIMAL,
     DUAL,
     NONE
}